### DISCUSSION TITLE
How To build the project in vscode

### DISCUSSION THREAD

**Discussion by upretyrohan** [jacktrip/jacktrip#529] [Howtos] [answered]
cmake:configure gives the error on linux vsvode
[main] Configuring folder: jacktrip 
[rollbar] Unhandled exception: Failed writing to file /home/omsp/jacktrip/build/.cmake/api/v1/query/client-vscode/query.json Error: Cannot create /home/omsp/jacktrip/build/.cmake: /home/omsp/jacktrip/build is a non-directory {}
[driver] Failed to configure project

**Comment by ntonnaett** [comment, marked as answer]
CMake is not supported. You can use Meson with a vscode **extension called mesonbuild**. There is a deprecated version from the original author. Make sure you choose the version from mesonbuild.

**Comment by ntonnaett** [comment]
@cchafe This is a question that would fit very well in a Discussion page. Could we enable this for jacktrip?
